docs/support-frontend-switches.md

```diff
@@ -2,10 +2,11 @@
 
 This app has a switchboard (at /switches) for enabling/disabling features on support.theguardian.com (support-frontend).
 
-The switches live in the support-admin-console S3 bucket, in `switches_v2.json`.
+The switches live in the support-admin-console S3 bucket, in `switches_v2.json`. Please donot copy `switches_v2.json` between Code, Dev, Prod because their settings may differ.
```

Comment:
I think it might be good to suggest diffing the files pre-upload, what do you think?

```suggestion
The switches live in the support-admin-console S3 bucket, in `switches_v2.json`. Be careful not to copy `switches_v2.json` between Code, Dev, and Prod because their settings may differ. Before uploading your changed version, consider downloading the current file and diffing it with your changed file, to make sure you only see the changes you expect.
```